Jakefile.js

```diff
@@ -924,7 +924,8 @@ function lintFileAsync(options, path, cb) {
 var lintTargets = compilerSources
     .concat(harnessCoreSources)
     .concat(serverCoreSources)
-    .concat(scriptSources);
+    .concat(scriptSources)
+    .concat([path.join(servicesDirectory,"services.ts")]);

```

Comment:
It is currently only services file as to fix entire services folder for linting will be a large fix